mlflow/server/js/src/model-registry/components/ModelVersionView.js

```diff
@@ -316,8 +316,15 @@ export class ModelVersionViewImpl extends React.Component {
         </a>
       );
     } else if (runInfo) {
+      const artifactPath = modelSource.substring(modelSource.indexOf('/artifacts/') + 11);
```

Comment:
- please fix lint issues
- let's assert that `modelSource` exists
- `11` seems to be a magic value here - what about `.+\/artifact\/(.+)$` regex wrapped in utility function to make it bit more elegant? ✨